app/views/components/_reorderable_list.html.erb

```diff
@@ -2,6 +2,8 @@
   items ||= []
   data_attributes ||= {}
   data_attributes[:module] = "reorderable-list"
+  data_gtm = data_attributes[:gtm] || "reorderable-list"
```

Comment:
I think we should avoid GTM attributes here, since they're coupled to our approach to analytics. We could just handle this as part of the listener, much like [we do for topics](https://github.com/alphagov/content-publisher/blob/master/app/assets/javascripts/modules/gtm-topics-listener.js#L3).

Reply:
I've simplified the way we send data to GTM (without requiring a data attribute)